openapi-v1.yaml

```diff
@@ -5287,6 +5320,80 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
+  /notebooks/{namespace}/{array}/copy:
```

Comment:
The draft PR for the rest server uses timestamps for versions. We should include them in the spec

Reply:
`end_timestamp` added